elements/pfe-styles/src/red-hat-font.scss

```diff
@@ -0,0 +1,119 @@
+/*    
```

Comment:
Can you use the /// sassdoc syntax so that we can add this to sassdoc eventually?  Great information to have.